blocks/api/raw-handling/is-inline-content.js

```diff
@@ -1,7 +1,56 @@
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
 /**
  * Internal dependencies
  */
-import { isInline, isDoubleBR } from './utils';
+import { isPhrasingContent } from './utils';
+
+/**
+ * An array of tag groups used by isInlineForTag function.
+ * If tagName and nodeName are present in the same group, the node should be treated as inline.
+ * @type {Array}
+ */
+const inlineWhitelistTagGroups = [
+	[ 'ul', 'li', 'ol' ],
+	[ 'h1', 'h2', 'h3', 'h4', 'h5', 'h6' ],
+];
+
+/**
+ * Checks if nodeName should be treated as inline when being added to tagName.
+ * This happens if nodeName and tagName are in the same group defined in phrasingContentTagGroups.
+ *
+ * @param {string} nodeName Node name.
+ * @param {string} tagName  Tag name.
+ *
+ * @return {boolean} True if nodeName is inline in the context of tagName and
+ *                    false otherwise.
+ */
+function isInlineForTag( nodeName, tagName ) {
```

Comment:
Is `nodeName` uppercase or lowercase here? It's confusing to me looking at `inlineWhitelistTagGroups` as lowercase, but knowing that `Element#nodeName` is uppercase.

Reply:
As noted above, I just moved these functions without changing them, and I'm not the author. No excuse though, I'll adjust these.